src/plugins/visualizations/public/embeddable/create_vis_embeddable_from_object.ts

```diff
@@ -40,12 +40,12 @@ import { IContainer, ErrorEmbeddable } from '../../../embeddable/public';
 import { DisabledLabEmbeddable } from './disabled_lab_embeddable';
 import {
   getSavedVisualizationsLoader,
-  getSavedAugmentVisLoader,
   getUISettings,
   getHttp,
   getTimeFilter,
   getCapabilities,
 } from '../services';
+import { getSavedAugmentVisLoader } from '../../../vis_augmenter/public';
```

Comment:
I'm ok with this change, but if we're going to fetch it from vis augmenter can you remove the `setSavedAugmentVisLoader` call in `visualizations/public/plugin.ts`?

Reply:
something like `setSavedAugmentVisLoader(visAugmenter.augmentVisSavedObjectLoader)` in the lifecycle function in `visualization` `public/plugin.ts` file

Reply:
I was getting issues before since it was not setting the getter and setter correctly and I think it made sense to use the correctly defined getter and setter which should be from `vis_augmenter`.

Reply:
See https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3961#discussion_r1192885266

Reply:
Updated now.